src/20240121_Multi_UpdateStETHAndWETHRiskParamsOnAaveV3EthereumOptimismAndArbitrum/UpdateStETHAndWETHRiskParamsOnAaveV3EthereumOptimismAndArbitrum.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Update stETH and WETH Risk Params on Aave v3 Ethereum, Optimism and Arbitrum"
```

Comment:
```suggestion
title: "Update ETH EMode and WETH Risk Params on Aave v3 Ethereum, Optimism and Arbitrum"
```

I would not use the asset name stETH specifically, as when updating the ETH correlated emode risk params - we update it for all the other assets in the emode category, for example: rETH, cbETH, rETH in case of ethereum v3 are also being affected (which should be included in the writeup depending on each network)

At other places, I would use the name wstETH instead of stETH (to avoid confusion as we do not have stETH listing on v3, we have it only on v2 market)

(similarly for other places as well)

Reply:
done, thx a lot.